schema/schema_test.go

```diff
@@ -463,15 +470,30 @@ func TestDefaults(t *testing.T) {
 		"def_str":{"type":"string","default":"str1"},
 		"def_str1":{"type":"string","default":"st'r2"},
 		"def_arr_str":{"type":"array","default":["one", "two"], "items": {"type":"string"}},
-		"def_obj_str":{"type":"object","default":{"Field1":"aaa"}, "properties": {"Field1" : { "type":"string" }}},
+		"def_obj_str":{
+			"type":"object",
+			"default":{"Field1":"aaa"},
+			"properties": {
+				"Field1" : { "type":"string" },
+				"field_nested_required":{"type":"integer"},
+				"field_nested_required2":{"type":"integer"},
+				"field_nested_index2":{"type":"integer"},
+				"field_nested_index_and_req":{"type":"integer"}
+			},
+			"index":["field_nested_index2","field_nested_index_and_req"],
```

Comment:
I'd say have it on the field level in the schema as well. All attributes then will be in the same place like sort, unique, etc.

Reply:
makes sense